armi/settings/fwSettings/globalSettings.py

```diff
@@ -572,12 +572,16 @@ def defineSettings() -> List[setting.Setting]:
             description="Number of blocks with control for a REBUS poison search",
         ),
         setting.Setting(
-            CONF_NUM_COUPLED_ITERATIONS,
-            default=0,
-            label="Tight Coupling Iterations",
-            description="Number of tight coupled physics iterations to occur at each "
-            "timestep",
-            schema=vol.All(vol.Coerce(int), vol.Range(min=0)),
+            CONF_TIGHT_COUPLING,
+            default=False,
+            label="Tight Coupling",
+            description="Boolean to turn on/off tight coupling",
+        ),
+        setting.Setting(
+            CONF_TIGHT_COUPLING_MAX_ITERS,
+            default=15,
```

Comment:
Seems high as a default. Maybe 4?

Reply:
Ah yeah, good call. @keckler suggested the same thing a while back.

Reply:
https://github.com/terrapower/armi/pull/1033/commits/b0debd17c3e9073a9ee2943215e867ae0a60ff90